manifests/server/puppetserver.pp

```diff
@@ -35,11 +49,22 @@
 #   }
 #
 class puppet::server::puppetserver (
-  $java_bin          = $::puppet::server_jvm_java_bin,
-  $config            = $::puppet::server_jvm_config,
-  $jvm_min_heap_size = $::puppet::server_jvm_min_heap_size,
-  $jvm_max_heap_size = $::puppet::server_jvm_max_heap_size,
-  $jvm_extra_args    = $::puppet::server_jvm_extra_args,
+  $java_bin                    = $::puppet::server_jvm_java_bin,
+  $config                      = $::puppet::server_jvm_config,
+  $jvm_min_heap_size           = $::puppet::server_jvm_min_heap_size,
+  $jvm_max_heap_size           = $::puppet::server_jvm_max_heap_size,
+  $jvm_extra_args              = $::puppet::server_jvm_extra_args,
+  $server_puppetserver_dir     = $::puppet::server_puppetserver_dir,
+  $server_jruby_gem_home       = $::puppet::server_jruby_gem_home,
+  $server_ruby_load_paths      = $::puppet::server_ruby_load_paths,
+  $server_cipher_suites        = $::puppet::server_cipher_suites,
+  $server_max_active_instances = $::puppet::server_max_active_instances,
+  $server_ssl_protocols        = $::puppet::server_ssl_protocols,
+  $server_ca                   = $::puppet::server_ca,
+  $server_dir                  = $::puppet::server_dir,
+  $vardir                      = $::puppet::vardir,
+  $server_ca_client_whitelist  = $::puppet::server_ca_client_whitelist,
+  $server_admin_api_whitelist  = $::puppet::server_admin_api_whitelist,
```

Comment:
There are some values missing, that will be needed in the template later. At least:
$server_idle_timeout
$server_connect_timeout
$server_enable_ruby_profiler